source/includes/_invoice.md

```diff
@@ -3583,9 +2125,6 @@ class CustomField {
    }
 ]
```

Comment:
Same comment as above, I think an empty body is sent, could you please confirm and update if required?